{"uuid": "d00dfe12-af8c-49da-a298-2a4296301685", "counter": 53, "stdout": "\u001b[0;31mfatal: [demo03]: FAILED! => {\"changed\": false, \"msg\": \"One or more supplied key could not be found in the database.\"}\u001b[0m\r\n\u001b[0;36m...ignoring\u001b[0m", "start_line": 57, "end_line": 59, "runner_ident": "04b27d39-be08-4d17-bea1-4ae5e9169435", "event": "runner_on_failed", "pid": 173462, "created": "2025-05-15T20:09:44.391970+00:00", "parent_uuid": "005056b7-f67d-1069-3d3c-000000000014", "event_data": {"playbook": "/tmp/tmprx9_fzyo.yml", "playbook_uuid": "ddfc03db-499d-4520-830b-1aac754be952", "play": "Instalaci\u00f3n y configuraci\u00f3n de Pure-FTPd para hosts", "play_uuid": "005056b7-f67d-1069-3d3c-000000000002", "play_pattern": "demo03", "task": "Verificar si el usuario FTP existe", "task_uuid": "005056b7-f67d-1069-3d3c-000000000014", "task_action": "getent", "resolved_action": "ansible.builtin.getent", "task_args": "", "task_path": "/tmp/tmprx9_fzyo.yml:142", "host": "demo03", "remote_addr": "demo03", "res": {"msg": "One or more supplied key could not be found in the database.", "invocation": {"module_args": {"database": "passwd", "key": "ftpuser", "fail_key": true, "service": null, "split": null}}, "_ansible_no_log": false, "changed": false}, "start": "2025-05-15T20:09:43.600462+00:00", "end": "2025-05-15T20:09:44.391586+00:00", "duration": 0.791124, "ignore_errors": true, "event_loop": null, "uuid": "d00dfe12-af8c-49da-a298-2a4296301685"}}